spark/src/test/scala/org/apache/comet/parquet/ParquetReadSuite.scala

```diff
@@ -434,8 +434,8 @@ abstract class ParquetReadSuite extends CometTestBase {
                   i.toFloat,
                   i.toDouble,
                   i.toString * 48,
-                  java.lang.Byte.toUnsignedInt((-i).toByte),
-                  java.lang.Short.toUnsignedInt((-i).toShort),
```

Comment:
Do you check if Spark also return these values?

Reply:
I would say that is covered by the `"basic data type support"` test

Reply:
I meant that you changed this expected results to match what Comet returns, but are they same as Spark returns?

Reply:
Yes the same as Spark